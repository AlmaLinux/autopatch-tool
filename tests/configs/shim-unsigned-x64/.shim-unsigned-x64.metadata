e6f506462069aa17d2e8610503635c20f3a995c3 SOURCES/redhatsecurebootca5.cer
cdec924ca437a4509dcb178396996ddf92c11183 SOURCES/shim-15.8.tar.bz2
