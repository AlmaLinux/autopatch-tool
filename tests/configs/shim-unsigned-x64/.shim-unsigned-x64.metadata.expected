cdec924ca437a4509dcb178396996ddf92c11183 SOURCES/shim-15.8.tar.bz2
